README.md

```diff
@@ -234,7 +234,7 @@ Your terminal will just hang there. This means the server is running. Any errors
 In the new terminal, we will now connect to the running database server using a client. Dolt also ships with a MySQL compatible client. 
 
 ```bash
-% dolt sql-client -u root
+% dolt -u root -p "" sql
```

Comment:
just say `dolt sql`

In the line before, you may want to clarify that they should run this in the same directory where they started `dolt sql-server`